### DISCUSSION TITLE
Is sandpack powered by Parcel2

### DISCUSSION THREAD

**Discussion by maxkoretskyi** [codesandbox/sandpack#881] [Q&A] [answered]
I'm wondering if Sandpack, specifically `sandpack-react` is powered by Parcel2? It'd be great to understand the relationship between Parcel2, sandpack toolkit and sandbox (hosting evn). Appreciate any insights

**Comment by DeMoorJasper** [comment]
Some parts of sandpack are inspired by how parcel works, but it does not running parcel itself, parcel itself is a bit too heavy to run in a browser unfortunately.

**Reply by maxkoretskyi** [reply]
thanks, so is there a stanalone version of a bundler that's used inside sandpack? I want to give it a try with Angular playground and plain HTML+JS+CSS, but would like to experiment with a standalone bundler first. It seems like the main focus is React based playgrounds, so want to test the water before I go all in on sandpack for other types of frameworks/setups

**Reply by DeMoorJasper** [reply]
Yes we have the vite templates, based on nodebox running the actual version of vite. You could use the node template and define the tooling you'd use locally in pkg.json and it should work

**Reply by maxkoretskyi** [reply]
cool! thanks for the good advice. It'll save me some time if you provide a bit more elaborate explanation 🤓. Specifically this part
> based on nodebox running the actual version of vite. You could use the node template...

what's `nodebox`? and should I use `node` template for `sandpack` like this:

```
<Sandpack template="node">
```

I have doubts b/c as I see it includes express server, not sure what I need it.

**Reply by danilowoz** [reply]
You can find more details what's Nodebox [here](https://codesandbox.io/blog/announcing-sandpack-2) on the announcement blog post. 

> I have doubts b/c as I see it includes express server, not sure what I need it.

The node template doesn't include any dependencies, it's a "vanilla" Node.js template where you can run any node script and install dependencies as needed.

**Reply by maxkoretskyi** [reply]
>  it's a "vanilla" Node.js template where you can run any node script

yeah, that's where I'm confused, I don't really need a "backend" server for Angular, it's all JS runtime env that just needs to be bundled. Maybe I need vanilla vite template instead? this seems to be a supported option.

 I thought a bit about this, so do you mean I need something like this
 
 ```
const express = require('express');
const app = express();

buildAngularAppIntoDistWithVite();

app.use(express.static('dist'));

app.listen(3000);
```

**Reply by danilowoz** [reply, marked as answer]
Well, I'm not familiar with Angular to express any opinion. But take into account that any tutorial that guides you how to bundle Angular on Node.js should work on Nodebox too, for example https://medium.com/@hiepxanh/angular-vite-example-is-crazy-fast-3ee4d730020c

**Reply by maxkoretskyi** [reply]
nice, thanks! let me give it a try and I'll come back with the feedback :)
